apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -144,3 +146,17 @@ type ProjectConfig struct {
 func init() {
 	SchemeBuilder.Register(&ProjectConfig{})
 }
+
+// Validate validates the controller configuration (ProjectConfig).
+func (c *ProjectConfig) Validate() error {
```

Comment:
Could we validate more stuff? Some suggestions:
* `.metrics.bindAddress` is a valid address
* `.health.healthProbeBindAddress` is a valida ddress
* `.webhook.port` is a valid port
* `.images[]` follow valid container name image schema

Reply:
I've added validation for images (part of our custom/additional operator settings). The other settings come from the controller-runtime project, and are validated there (the operator binary will print an error message on startup if they're invalid).